src/lib.rs

```diff
@@ -4,7 +4,7 @@
 #![warn(rust_2018_idioms)]
 
 //! Data-parallelism library that makes it easy to convert sequential
-//! computations into parallel
+//! computations into parallel.
```

Comment:
Well, it's not a proper sentence either -- maybe add "Rayon is a ..." to start?

Reply:
Sure. Done.